arviz/stats/diagnostics.py

```diff
@@ -996,3 +996,136 @@ def _multichain_statistics(ary, focus="mean"):
         ess_tail_value,
         rhat_value,
     )
+
+
+def psens(data, *, component, var_names=None, delta=0.01, dask_kwargs=None):
+    """Compute power-scaling sensitivity diagnostic.
+
+    Power-scales the prior or likelihood and calculates how much the posterior is affected.
+
+    Parameters
+    ----------
+    data : obj
+        Any object that can be converted to an :class:`arviz.InferenceData` object.
+        Refer to documentation of :func:`arviz.convert_to_dataset` for details.
+        For ndarray: shape = (chain, draw).
+        For n-dimensional ndarray transform first to dataset with ``az.convert_to_dataset``.
+    var_names : list
+        Names of variables to include in the rhat report
+    component : str
+        Select component to power-scale. Valid components are are:
+        - "prior"
+        - "likelihood"
+    dask_kwargs : dict, optional
+        Dask related kwargs passed to :func:`~arviz.wrap_xarray_ufunc`.
+
+    Returns
+    -------
+    xarray.Dataset
+      Returns dataset of power-scaling sensitivity diagnostic values.
+
+    Notes
+    -----
+    The diagnostic is computed by power-scaling the specified component (prior or likelihood)
+    and determining the degree to which the posterior changes. It uses Pareto-smoothed
+    importance sampling to avoid refitting the model.
+
+    References
+    ----------
+    * Kallioinen et al. (2022) see https://arxiv.org/abs/2107.14054
```

Comment:
```suggestion
    .. [1] Kallioinen et al, *Detecting and diagnosing prior and likelihood sensitivity with 
       power-scaling*, 2022, https://arxiv.org/abs/2107.14054
```

The `.. [1]` part is needed for the other [1] above to be a link. The slight style change is to make it a bit more similar to numpy's docs, for example [eigh](https://numpy.org/doc/stable/reference/generated/numpy.linalg.eigh.html#rc702e98a756a-1), but feel free to change it if you prefer a different citation style, the numpydoc style guide (https://numpydoc.readthedocs.io/en/latest/format.html#references) only covers the [1] style links, not the reference format.